### PR TITLE
[14.0][NEW] hr_timesheet_purchase_order

### PR DESCRIPTION
**Pull request by ivs-cetmix**
Add new module to create purchase order from timesheets

### CODE REVIEW

File: hr_timesheet_purchase_order/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+============================================
+Create purchase orders from timesheet sheets
+============================================
+
+This module allows you to create Purchase Orders based on the employee timesheet.
```
Comment (francesco-ooops) [inline hr_timesheet_purchase_order/readme/DESCRIPTION.rst]: *based on the employee timesheet sheet (see module: https://github.com/OCA/timesheet/tree/14.0/hr_timesheet_sheet )

File: hr_timesheet_purchase_order/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Add a *Billing partner* in the *HR* tab of the *Employee* form and enable the *Purchase Order* checkbox
```
Comment (francesco-ooops) [inline hr_timesheet_purchase_order/readme/USAGE.rst]: * Go to *_Employees_* app > select an employee > go to *_HR Settings_* tab and enable the *_Generate Purchase Order from timesheet sheet_" checkbox
* Select the *_Billing partner_* for which the PO will be created

File: hr_timesheet_purchase_order/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+* Go to Employees app > select an employee > go to HR Settings tab and enable the "Generate Purchase Order from timesheet sheet" checkbox
+* Select the Billing partner for which the PO will be created
+* Click the *Create Purchase Order* in the  *Timesheet Sheet* form to create a new RFQ
```
Comment (francesco-ooops) [inline hr_timesheet_purchase_order/readme/USAGE.rst:3]: A server action to create POs is also available in tree view.

**Review by francesco-ooops — APPROVED** [review]
From description: 

> This module allows you to create Purchase Orders based on the employee timesheet sheet.
This might be useful for subcontrating and outsourcing.

**Review by elvise — APPROVED** [review]
LGTM!

**Comment by elvise** [comment]
@pedrobaeza what do you think ?

**Comment by pedrobaeza** [comment]
Please change commit message to `[ADD] hr_timesheet_purchase_order: Add new module to create purchase order from timesheets` (no version in it - the version is put in the PR title - and the name of the module included).

**Comment by elvise** [comment]
> Please change commit message to `[ADD] hr_timesheet_purchase_order: Add new module to create purchase order from timesheets` (no version in it - the version is put in the PR title - and the name of the module included).

Sure! @isserver1 please follow this comment

**Comment by elvise** [comment]
@pedrobaeza is it good now ?

**Pull request merged**